src/Transfer/Adapter/FilterPluginManager.php

```diff
@@ -20,16 +21,19 @@
  */
 class FilterPluginManager extends BaseManager
 {
-    /**
-     * Default set of filters
-     *
-     * @var array
-     */
-    protected $aliases = [
-        'decrypt'   => 'filedecrypt',
-        'encrypt'   => 'fileencrypt',
-        'lowercase' => 'filelowercase',
-        'rename'    => 'filerename',
-        'uppercase' => 'fileuppercase',
-    ];
+
+    public function __construct($configOrContainerInstance = null, array $v3config = []) 
```

Comment:
@Ocramius that's the naming current under zend-servicemanager; in v3, it's a configuration array; v2 doesn't accept a second argument. Hence, `$v3config`. (Hey, it's not great, but I got a green light on review for that...)